Core/HLE/sceFont.cpp

```diff
@@ -747,6 +915,12 @@ void __FontDoState(PointerWrap &p) {
 	__KernelRestoreActionType(actionPostAllocCallback, PostAllocCallback::Create);
 	p.Do(actionPostOpenCallback);
 	__KernelRestoreActionType(actionPostOpenCallback, PostOpenCallback::Create);
+	p.Do(actionPostOpenAllocCallback);
```

Comment:
Need to update `auto s = p.Section("sceFont", 1);` to `1, 2` and check that `s >= 2`.

In the other case, it gets trickier.  The simplest thing would be to have a flag like `static bool useAllocCallbacks = true;` and set it to false if the save state is old.  Then we'd keep not using those callbacks, and wouldn't need to worry about restoring them safely.

-[Unknown]